e2e/ctl_v3_alarm_test.go

```diff
@@ -99,6 +108,101 @@ func alarmTest(cx ctlCtx) {
 	}
 }
 
+type fakeConsistentIndex struct{ rev uint64 }
+
+func (f *fakeConsistentIndex) ConsistentIndex() uint64 { return f.rev }
+
+func alarmCorruptTest(cx ctlCtx) {
```

Comment:
why isn't this an integration test? there's already a corruption test there

Reply:
If we fatal in corruption, fatal logger calls `os.Exit(1)` (so not testable)?